hack/setup-build-index-image.sh

```diff
@@ -54,8 +54,10 @@ spec:
           wget https://github.com/operator-framework/operator-registry/releases/download/v1.23.0/linux-amd64-opm
           mv linux-amd64-opm opm
           chmod +x ./opm
+          set +x
           export pass=$( jq .\"image-registry.openshift-image-registry.svc:5000\".password /var/run/secrets/openshift.io/push/.dockercfg )
```

Comment:
Should this also not be 'export'?  If the only use is the next line, a non-exported variable would be safer, otherwise every command after this point may have access to it and accidentally log it if they log their environment.

Reply:
Yeah, good catch. Let me do it and test.